packaging/rpm/amanda.spec.src

```diff
@@ -218,8 +119,13 @@ BuildRequires: %{curl}-devel >= 7.10.0
 BuildRequires: openssl
 BuildRequires: openssl-devel
 BuildRequires: perl(ExtUtils::Embed)
-BuildRequires: mailx
-%{?prereq_sharutils}
+BuildRequires: ncurses-devel
+#
+# NOTE: keep Requires with installable packages only...
+#
+%package backup_client
```

Comment:
Keep all the Requires separate from the BuildRequires... as the above is not installed.